fili-core/src/main/java/com/yahoo/bard/webservice/async/ResponseException.java

```diff
@@ -64,7 +95,29 @@ public ResponseException(
             DruidQuery<?> druidQuery,
             Throwable cause
     ) {
-        super(buildMessage(reason, description, statusCode, druidQuery, cause));
+        this(statusCode, reason, description, druidQuery, cause, new ObjectMapper().writer());
+    }
```

Comment:
Oh, and also not increase the number of deprecation warnings the compiler throws at us if we can 😉